frontend/src/actions/commonActions.js

```diff
@@ -173,3 +91,18 @@ export const getRequestParams = (type) => (dispatch, getState) => {
 
   return params;
 };
+
+export const calculateSummary = (countObj) => {
+  const total = Number(countObj["total"]) || 0;
```

Comment:
Should this be `countObj?.total || 0`?? Or ... if you're sure that `total` exists, why wouldn't it be an integer value already: and if it might *not* exist, the conditional fetch seems safer.

Reply:
Yeah right. In some cases, the values in countObj were strings, which led to issues like NaN or unexpected string concatenation during arithmetic operations.